pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -683,28 +683,28 @@ func (ku *KubeUtil) setKubeletHost(hosts *connectionInfo, httpsPort, httpPort in
 		connectionErrors = append(connectionErrors, errors...)
 	}
 
-	log.Error("All connection attempts to the Kubelet failed.")
+	log.Debugf("All connection attempts to the Kubelet failed.")
```

Comment:
As we don't have string formating tokens, `log.Debug` is enough here

Reply:
thanks, done 👍